components/raftstore/src/store/fsm/peer.rs

```diff
@@ -747,7 +748,7 @@ where
         syncer: UnsafeRecoveryExecutePlanSyncer,
         failed_voters: Vec<metapb::Peer>,
     ) {
-        if self.fsm.peer.unsafe_recovery_state.is_some() {
```

Comment:
Can we keep the variable name? Seems it's another function.

Reply:
I think mixing unsafe recovery and snapshot recovery isn't a good idea.

Reply:
fixed